libraries/freertos_plus/standard/crypto/src/iot_crypto.c

```diff
@@ -263,6 +278,101 @@ void CRYPTO_ConfigureThreading( void )
                                aws_mbedtls_mutex_unlock );
 }
 
+
+void CRYPTO_ConfigureDRBG( void )
```

Comment:
Is there always exactly one context?

Reply:
Yes, in the CRYPTO layer I am going to maintain only 1 DRBG that all other applications/libraries and request data from.